test/mt_id_test.c

```diff
@@ -1,5 +1,6 @@
 #include "h5test.h"
 #include "H5Iprivate.h"
+#include "testhdf5.h"
```

Comment:
Is there a reason to include `testhdf5.h` here?

Reply:
There is not a reason to include that. I forgot to remove it while trying to debug the not printing issue with testing.